scipy/special/tests/test_data.py

```diff
@@ -489,11 +490,8 @@ def test_local():
         data_local(ellip_harm, 'ellip',(0, 1, 2, 3, 4), 5, rtol=1e-10, atol=1e-13),
     ]
 
-    with warnings.catch_warnings():
-        warnings.simplefilter("ignore", category=IntegrationWarning)
-
-        for test in TESTS:
-            yield _test_factory, test
+    for test in TESTS:
+        yield _test_factory, test
```

Comment:
Here is again some difference to pytest: with it, it was necessary to decorate the ellip_harm_2, ellip_harm functions above and sph* functions below.

The previous differences were also in yield tests...